.gitignore

```diff
@@ -34,3 +34,4 @@ vignettes/*.pdf
 
 # Shiny token, see https://shiny.rstudio.com/articles/shinyapps.html
 rsconnect/
+.Rproj.user
```

Comment:
```suggestion
```
I think this is already ignored further up this file, so it should not be necessary here.